reference/5.0/PackageManagement/Uninstall-Package.md

```diff
@@ -213,7 +225,9 @@ Accept wildcard characters: False
 
 ### -InputObject
 
-Specifies a package by using the package's SoftwareIdentity type, which is shown in the results of the Get-Package cmdlet.
+Accepts pipeline input that specifies the package's **SoftwareIdentity** object from the
+`Get-Package` dmdlet. **InputObject** accepts the **SoftwareIdentity** object as a `Get-Package`
```

Comment:
Typo: dmdlet

Reply:
Thanks! Fixed and uploaded changed files.